frontend/components/ui/blocknote/editor/custom-blocknote.css

```
@@ -85,3 +85,121 @@
+  .bn-block[data-selected="true"]
+  [data-content-type="latex"]
+  *,
+.bn-container.light
+  .bn-block[data-selected="true"]
+  [data-content-type^="latex"],
+.bn-container.light
+  .bn-block[data-selected="true"]
+  [data-content-type^="latex"]
+  * {
+  background-color: transparent !important;
+  background: transparent !important;
```

Biome (2.1.2)

[error] Unexpected shorthand property background after background-color



(lint/suspicious/noShorthandPropertyOverrides)